src/Resources/Locales/en.json

```diff
@@ -1392,11 +1395,20 @@
                 "messages": {
                     "noColor": "Not set",
                     "loading": "Loading historized data...",
-                    "noData": "No historized data found in selected time frame.",
+                    "noData": "No historized data found in the selected time range.",
                     "noSeries": "No twin selected to query, please add twin properties to view historized data."
                 }
             },
             "noSeriesDescription": "To view data history for this twin, select a property to view."
         }
+    },
+    "dataHistoryErrorHandlingWrapper": {
+        "errors": {
+            "genericNetwork": "Cluster cannot be connected which may be due to invalid url, stopped service or a network error.",
```

Comment:
suggestion: `Cluster cannot be reached. This may be due....`

Reply:
changed to "Azure Data Explorer instance cannot be connected, make sure the resource is not stopped and your network connectivity is stable." from content review by Azima and Riley